testing/src/main/java/com/yubico/yubikit/testing/fido/Ctap2SessionTests.java

```diff
@@ -98,8 +99,16 @@ private static void doTestCancelCborCommand(Ctap2Session session, boolean delay)
 
         try {
             session.makeCredential(
-                    TestData.CLIENT_DATA_HASH, TestData.RP.toMap(), TestData.USER.toMap(), Collections.singletonList(TestData.PUB_KEY_CRED_PARAMS_ES256.toMap()),
-                    null, null, null, pinAuth, 1, state);
+                    TestData.CLIENT_DATA_HASH,
+                    TestData.RP.toMap(),
+                    TestData.USER.toMap(SerializationType.CBOR),
+                    Collections.singletonList(TestData.PUB_KEY_CRED_PARAMS_ES256.toMap()),
```

Comment:
The data we send to makeCredential should be CBOR maps, so I would explicitly provide SerializationType.CBOR to all these toMap calls, even if the JSON Maps in this case are identical.

Reply:
fixed